addons/stock/models/stock_orderpoint.py

```diff
@@ -0,0 +1,158 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from datetime import datetime, time
+from dateutil import relativedelta
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError, ValidationError
+from odoo.osv import expression
+
+
+class Orderpoint(models.Model):
```

Comment:
not sure if it's the right commit but i would like to rename this class StockWarehouseOrderpoint :/